jwst/exp_to_source/exp_to_source.py

```diff
@@ -7,8 +7,8 @@
 
 from ..datamodels import (
     MultiExposureModel,
-    SourceModelContainer
-)
+    SourceModelContainer,
+    DataModel)
```

Comment:
Due to the most recent commit `DataModel` is no longer used and hence flake8 is throwing an error in the Travis tests. Fix this and it should be good to go.